client/imports/app/modules/core/csvjsonparse/csvjson.component.ts

```diff
@@ -304,8 +321,23 @@ export class CsvJsonComponent implements OnInit, OnDestroy {
 		});
 		this.uploadresult["updatedstring"] = dummyfeed;
 	}
+	lastupdateddata() {
+	this.csvData = Csvdata.find({}, {
+		sort: {
+			"lastUpdated": -1
+		}
+	})
+	this.csvSub = MeteorObservable.subscribe('csvdata').subscribe();
+	this.csvData.subscribe((data) => {
+	this.lastUpdate = true;
+	this.latestdata = [data[0]];
```

Comment:
why is [data[0]] needed? explain

Reply:
sir I am sorting  the data on basis of last updated date and pushing the 0th index data to it.

Reply:
what is the exact logic you have used. where are u storing the data?

Reply:
this.csvData = Csvdata.find({}, {
		sort: {
			"lastUpdated": -1
		}
	})

Reply:
but csv data has transaction data. i.e every row of csv. u r storing file information also there?

Reply:
it doesn't make sense to store file in transaction data. beacuse we can upload multiple transaction files and it will overwrite the previous entry. so this will not working properly. think about this properly. u need to create a new collection